wordpress-mobile/WordPress-iOS
# Issue/aztec media insertion

This PR adds support to insert media on a post from the device or the media library.

Note: this PR doesn't support upload progress or check to see if media is still going on when closing post that will come on separate PRs.

To test:
 - Open the app
 - Activate the Aztec Editor on Me->Settings
 - Create a new Post
 - Add Media to it from the device or from the WordPress Media Library
 - Check if upload is done correctly

Needs review: @diegoreymendez
